lib/routes/users.js

```diff
@@ -42,6 +42,73 @@ router.post('/', async (request, response) => {
   }
 });
 
+router.post('/login', async (request, response) => {
+  const {
+    body: { username, password },
+  } = request;
+  try {
+    client = new MongoClient(process.env.DB_CONNECTION, {
```

Comment:
See https://github.com/neuefische/cgn/issues/57#issuecomment-617603406

> Your API server should always have an active connection to the database. It's not recommended to connect in your API route handlers.

Thus, you should connect to your database on the server start (server.js?). Otherwise, your API calls would be much slower than required. Connecting to a database takes time.